base/applications/notepad/lang/ro-RO.rc

```diff
@@ -133,21 +133,21 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 CAPTION "Salt la poziție"
 BEGIN
-    LTEXT "Număr linie:", 0x155, 5, 12, 41, 12
+    LTEXT "Număr de linie:", 0x155, 5, 12, 41, 12
```

Comment:
width 41 might be a bit too narrow to make the longer text fit, but that is just my gutfeeling. I am not sure about that. A preview with ResourceHacker should bring you clarity. I will not withdraw my approval, because I am not certain in this case.

Reply:
![2024-03-08 01_01_46-](https://github.com/reactos/reactos/assets/113553683/b846367e-dcb9-4de0-b02f-32f6af96f873)

This is XP's DLL version.

Reply:
Yes, but as you can see: XP uses width 47 and not width 41. So that doesn't prove anything. You can build the ros version also with RosBE and drag it into ResourceHacker to be able to preview. I still believe this line might be problematic.